pkg/ai/mcp/backplane_console.go

```diff
@@ -39,34 +45,100 @@ func BackplaneConsole(ctx context.Context, request *mcp.CallToolRequest, input B
+	r, w, _ := os.Pipe()
+	os.Stdout = w
 
-	if err != nil {
-		errorMessage := fmt.Sprintf("Failed to access console for cluster '%s'. Error: %v", clusterID, err)
+	// Channel to capture the command execution result
+	resultChan := make(chan error, 1)
 
-		return &mcp.CallToolResult{
-			Content: []mcp.Content{
-				&mcp.TextContent{Text: errorMessage},
-			},
-		}, struct{}{}, nil // Return nil error since we're handling it gracefully
+	// Run the console command in a goroutine
+	go func() {
+		resultChan <- consoleCmd.RunE(consoleCmd, args)
+	}()
+
+	// Capture output with timeout to get the console URL
+	outputChan := make(chan string, 1)
+	go func() {
+		var buf bytes.Buffer
+		scanner := bufio.NewScanner(r)
+		for scanner.Scan() {
+			line := scanner.Text()
+			buf.WriteString(line + "\n")
+			// Once we see the console URL, we have what we need
+			if strings.Contains(line, "Console is available at") {
+				outputChan <- buf.String()
+				return
+			}
+		}
+		outputChan <- buf.String()
+	}()
+
+	var output string
+	var consoleURL string
+
+	// Wait for output with timeout
+	select {
+	case output = <-outputChan:
+		// Got the console URL
+	case <-time.After(30 * time.Second):
+		// Timeout waiting for console URL
+		output = "Console is starting (timeout waiting for URL)..."
+	}
+
+	// Restore stdout
+	w.Close()
+	os.Stdout = oldStdout
+	io.Copy(io.Discard, r) // Drain the pipe
+
+	// Check for immediate errors (non-blocking)
+	select {
+	case err := <-resultChan:
+		if err != nil {
+			return &mcp.CallToolResult{
+				Content: []mcp.Content{
+					&mcp.TextContent{Text: fmt.Sprintf("Failed to start console for cluster '%s'. Error: %v", clusterID, err)},
+				},
+			}, nil, nil
+		}
+	default:
+		// Command is still running in background - this is expected
+	}
+
+	// Extract console URL from output
+	// The actual output format is: "== Console is available at http://127.0.0.1:PORT =="
+	urlRegex := regexp.MustCompile(`Console is available at (https?://[^\s=]+)`)
+	matches := urlRegex.FindStringSubmatch(output)
+	if len(matches) > 1 {
+		consoleURL = matches[1]
 	}
 
 	// Build success message
 	var successMessage strings.Builder
-	successMessage.WriteString(fmt.Sprintf("Successfully accessed cluster console for '%s'", clusterID))
+	successMessage.WriteString(fmt.Sprintf("Successfully started cluster console for '%s'", clusterID))
+
+	if consoleURL != "" {
+		successMessage.WriteString(fmt.Sprintf("\n🔗 Console URL: %s", consoleURL))
+	}
 
 	if input.OpenInBrowser {
 		successMessage.WriteString("\n🌐 Console opened in default browser")
 	}
 
+	successMessage.WriteString("\n\nℹ️  Console is running in the background. It will continue to run until manually stopped.")
+
+	if output != "" {
+		successMessage.WriteString(fmt.Sprintf("\n\nInitial output:\n%s", strings.TrimSpace(output)))
+	}
+
 	return &mcp.CallToolResult{
 		Content: []mcp.Content{
 			&mcp.TextContent{Text: successMessage.String()},
 		},
-	}, struct{}{}, nil
+	}, nil, nil
```

Comment:
This may not align with other MCP endpoint returns

Reply:
> This may not align with other MCP endpoint returns

That's for debugging purposes, as I got test outcome:
```
I'll login to the specified cluster for you.

● backplane - login (MCP)(clusterId: "2eigjr6f5uj8u7l6ie5ql5656ohk1nbr")
  ⎿  {}

Access the console for cluster 2eigjr6f5uj8u7l6ie5ql5656ohk1nbr Don't open the browser

● I'll access the console for the specified cluster without opening the browser.

● backplane - console (MCP)(clusterId: "2eigjr6f5uj8u7l6ie5ql5656ohk1nbr", openInBrowser: false)
  ⎿  {}
```
So, can you clarify a bit why you used `struct{}` as Out type and returned `struct{}{}` in the initial PR-801? Looks the MCP SDK marshals it to `JSON "{}"` and overwrites the content field you set.  Can you please also test from your end to see if it return empty?